### PR TITLE
Fix/7479 Add missing padding above the header image

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
Add missing padding above the header image on RAT Profile info screen
- As requested, add missing padding on test certificates screens as well
## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7479

## Screenshots
Padding 7479

Test Certificate

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestCertificate/ExposureSubmissionTestCertificateViewModel.swift [hunk 1]
```diff
@@ -35,6 +35,17 @@ class ExposureSubmissionTestCertificateViewModel {
...
 		DynamicTableViewModel([
+			
+			// Add separator padding before the header image
+			.section(
+				header:
+					.separator(
+						color: .enaColor(for: .hairline)
+						),
+					cells: [
+					.space(height: 15)
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestCertificate/ExposureSubmissionTestCertificateViewModel.swift:46]: indent

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Consent/HealthCertificateInfoViewModel.swift [hunk 2]
```diff
@@ -24,6 +24,16 @@ struct HealthCertificateInfoViewModel {
...
 		DynamicTableViewModel([
 
+			// Add separator padding before the header image
+			.section(
+				header:
+					.separator(
+						color: .enaColor(for: .hairline)
+						),
+					cells: [
+					.space(height: 15)
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Consent/HealthCertificateInfoViewModel.swift:34]: indent

**Review by flxschmidt — APPROVED** [review]
LGTM :)

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestCertificate/ExposureSubmissionTestCertificateViewModel.swift (hunk 1 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestCertificate/ExposureSubmissionTestCertificateViewModel.swift:46]: It's fine on Xcode.

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Consent/HealthCertificateInfoViewModel.swift (hunk 2 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Consent/HealthCertificateInfoViewModel.swift:34]: This is also fine on Xcode.

**Review by 30mar — CHANGES_REQUESTED** [review]
This light gray line looks strange to me, im not sure if they updated the designs in figma or what, can you please provide the figma link so we can compare ?

**Comment by puneetmahali1** [comment]
> 
Actually, seems like there is a header space in iOS 12.5 and the bug assuming same in above iOS 12.5 too but let me check the figma designs with the colleagues.

**Pull request closed without merging**